### ISSUE TITLE
Update build for fat binary + arm64 support

### ISSUE THREAD

**Issue by jerrymarino** [MobileNativeFoundation/index-import#56]
At the time of writing, the latest release is cut for macOS x86_64

I'd like to run this binary on a M1 apple device, flagging this as a TODO for M1 support

**Comment by timsutton** [comment]
I built a universal one here: https://github.com/timsutton/index-import/releases/tag/5.3.2.6

It uses the same original x86_64 binary from this repo's release, and I lipo'ed in a new ARM build from same tag of this repo and the same swift release. The swift / libIndexStore build was all new to me (the instructions were very helpful!)

I thought potentially one could take this universal zip and either attach it as another binary asset to the original release in this repo. This would make it possible to then update rules_swift pointing to the universal asset instead of the x86_64-only one.

**Comment by jerrymarino** [comment]
@timsutton any way you'd want to consolidate this into a github action?

The repo has been generally working with `git clone && ./utils/update_checkout` - I think probably just have to set a specific target then lipo together the binary..

**Comment by timsutton** [comment]
Setting up a GH action for this seems like a good idea in general, would propose making a separate PR if folks are interested.

I wonder how frequently a cadence is useful here for actually building a new version? i.e. have do a build + release on any new swift tag that's discovered in the repo, or just wait for a new tag or version file to get bumped in this repo via a PR, in order to trigger a build of the same swift release tag version, for example?

**Comment by timsutton** [comment]
@keith Just wanted to bump this, if there's interest to at least get this universal zip added to this repo's releases? I've just gone through checking that it seems to be working for updating indexes on both Intel and an M1.

**Comment by keith** [comment]
I think setting up a github action for releases would be great, especially since we do it so infrequently. I've updated the release instructions [here](https://github.com/MobileNativeFoundation/index-import/pull/61) to make it even easier to build a fat binary for this case, I have no idea why I didn't create a new release then after I wrote them the first time.

I've just pushed a [new release](https://github.com/MobileNativeFoundation/index-import/releases/tag/5.5) built off the Swift 5.5 release branch. Realistically I assume there were not many changes, but it would be great if folks could update and report any potential issues!
